code/game/objects/items/ego_weapons/non_abnormality/limbus_ego.dm

```diff
@@ -43,3 +43,46 @@
 				heal_amt = 0
 		user.adjustBruteLoss(-heal_amt)
 		..()
+
+/obj/item/ego_weapon/mini/crow
+	name = "Crow's Eye View"
+	desc = "Now, break this birdcage... and fly free."
+	special = "This weapon attacks very fast. Use this weapon in hand to rush towards where you click."
+	icon_state = "crow"
+	icon = 'icons/obj/limbus_weapons.dmi'
+	inhand_icon_state = 'shiv'
```

Comment:
The checks are failing because this is supposed to be a "shiv" rather than 'shiv'
```suggestion
	inhand_icon_state = "shiv"
```